Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyFragment.kt

```diff
@@ -56,20 +64,34 @@ class SurveyFragment : Fragment() {
             )
             setContent {
                 JetsurveyTheme {
-                    viewModel.uiState.observeAsState().value?.let { surveyState ->
-                        when (surveyState) {
+                    val state = viewModel.uiState.observeAsState().value ?: return@JetsurveyTheme
```

Comment:
Why the `?: return@JetsurveyTheme` change?

Reply:
I needed a non-nullable state, and I generally prefer early returns instead of wrapping content in lets 😄  https://betterprogramming.pub/are-early-returns-any-good-eed4b4d03866